docs/api-reference/next.config.js/rewrites.md

```diff
@@ -276,31 +276,6 @@ module.exports = {
 }
 ```
 
-### Incremental adoption of Next.js
```

Comment:
I think we should leave this section and mention the new `fallback` rewrites more explicitly here unless this is too redundant 🤔  cc @leerob